### PR TITLE
HTC documentation review

### PR DESCRIPTION
**Pull request by andrea-manzi**
# Summary 
Review of the HTC documentation

----

**Related issue :**

### CODE REVIEW

File: content/en/users/htc/_index.md
Comment (gwarf) [inline content/en/users/htc/_index.md:105]:
```suggestion
  in the EGI infrastructure are [HTCondor-CE](https://htcondor-ce.readthedocs.io/en/latest/) 
```

Comment (gwarf) [reply content/en/users/htc/_index.md:105]: Sorry, I've got a problem with double/multiple spaces :)

File: content/en/users/htc/_index.md
Comment (andrea-manzi) [inline content/en/users/htc/_index.md:20]:
```suggestion
the infrastructure, the user can collect the output from the services that have
```

**Review by enolfc — COMMENTED** [review]
Now that you are touching this, should we move to high-throughput-compute instead of htc?

**Review by CatalinCondurache — APPROVED** [review]
It looks fine.

**Comment by andrea-manzi** [comment]
i just renamed the folder

**Review by gwarf — APPROVED** [review]
LGTM

**Comment by gwarf** [comment]
Hi @CatalinCondurache so it's on you to give the final review, thanks!
Reviews are currently voided when there are new changes, it's something we could also choose to change to ease validating changes by others, if people agree.

**Comment by gwarf** [comment]
> Reviews are currently voided when there are new changes

I've switched off the option that dismiss pull request reviews when new commits are pushed. It will be on us to decide if it's worth to ask for a formal re-review or if changes have just been addressed and PR can be merged.
It should add some more flexibility and avoid asking for a re-review for a trivial fix.

**Review by enolfc — CHANGES_REQUESTED** [review]
I know I'm a PITA, but now we need to change the CODEOWNERS file...

**Comment by gwarf** [comment]
It seems there are also some `.DS_Store` files, that needs to be cleaned. Also worth to be added to the `.gitignore`.
What should be changed for the `CODEOWNERS` ?

**Comment by enolfc** [comment]
> What should be changed for the `CODEOWNERS` ?

htc -> high-throughput-compute

**Comment by andrea-manzi** [comment]
should be ok now

**Review by enolfc — APPROVED** [review]
💯

**Review by CatalinCondurache — APPROVED** [review]
OK for now, although I believe we were thinking to be more consistent on using 'High-Throughput Compute' against 'HTC' everywhere in docs.

**Comment by gwarf** [comment]
@andrea-manzi would you want to replace things like the `EGI HTC` by `EGI High Throughput Compute`?
Indeed it probably makes more sense to use the expanded service name, while for the technology/concept itself HTC could be fine.

**Comment by andrea-manzi** [comment]
sure going to update the text

**Comment by gwarf** [comment]
Thanks!
I feel it's good enough to be merged, but I wonder if in the future we should try to have an habit to point to the service page like
```markdown
[EGI High Throughput Compute](https://www.egi.eu/services/high-throughput-compute/)
```
Opinions?

**Pull request merged**
